.github/workflows/test.yml

```diff
@@ -12,6 +12,7 @@ env:
   KIND_CLUSTER_NAME: ovn
   KIND_INSTALL_INGRESS: true
   KIND_ALLOW_SYSTEM_WRITES: true
+  PARALLEL: true
```

Comment:
some background here:  initially we tried to run in parallel, but that caused a lot of flakes (around 6 months ago). We may have fixed a bunch of the race condition bugs in the code so that now this is OK.

Reply:
yeah, now is configured to run only 4 parallel ginkgo process to avoid too much contention and races